addons/website_sale/views/templates.xml

```diff
@@ -921,7 +921,7 @@
                                                     </div>
                                                 </div>
                                             </div>
-                                            <div class="form-group col-12 s_website_form_submit s_website_form_no_submit_options">
+                                            <div class="form-group col-12 s_website_form_submit s_website_form_no_submit_options mb32">
```

Comment:
here I added the margin on that DIV because s_website_form_result could be the last element